### ISSUE TITLE
Use GitHub actions to automatically create issues for dependabot alerts

### ISSUE THREAD

**Issue by jakemer** [justifi-tech/web-component-library#215]
Use GitHub actions to automatically create issues when dependabot finds security vulnerabilities, and add the issue to our GitHub project board:

https://docs.github.com/en/code-security/dependabot/working-with-dependabot/automating-dependabot-with-github-actions

**Comment by colganmeanor** [comment]
I think I may have a path forward for this, but there is a small caveat. 

I found this article where someone created a a workflow in their GH repo that will create an issue whenever Dependabot _opens a PR_: [Create Issues Automatically with Dependabot
](https://larry.claman.net/post/2020-11-15-dependabot/)

This workflow ^^ basically listens for `pull_request` events in the repo and opens an issue IF `dependabot` is the user that opens the PR. 

It seems that might be the criteria we have to go by if we want a version of this action in our workflows - looking at [the list of events that trigger workflows ](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows), we are unable to listen to events related to `dependabot` (its not listed as an event we can listen for) - so our next best option would be to listen for _Pull Requests opened by Dependabot_.

We would need to also add a workflow that listens for new issues and automatically places them in the project board. But there are examples of that: [](https://github.com/actions/add-to-project)

@jakemer @TiagoFuelber what do you think of doing it this way? 

TLDR: We can add a workflow for this, but it will only be able to run when Dependabot opens a PR on its own. We will not be able to make a new issue for any Dependabot alert because Github does not allow us to listen to Dependabot events via their API.
